rasa/shared/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -484,8 +485,12 @@ def process_training_examples_by_key(
             if intent_metadata:
                 intent[KEY_METADATA] = intent_metadata
 
-            render_as_objects = any(KEY_METADATA in ex for ex in converted)
-            if render_as_objects:
+            examples_have_metadata = any(KEY_METADATA in ex for ex in converted)
+            example_texts_have_escape_chars = any(
+                has_string_escape_chars(ex.get(KEY_INTENT_TEXT, "")) for ex in converted
```

Comment:
optional: should it be `has_string_escaped_chars`?

Reply:
These are actually called escape characters :) https://en.wikipedia.org/wiki/Escape_character

Reply:
Isn't the `/` the "escape character" and something like `\n` is an "escape sequence"?